Dockerfile

```diff
@@ -14,9 +14,10 @@
 # limitations under the License.
 
 # Builds a docker image for Confluent's example applications for the Kafka Streams API
-ARG DOCKER_REGISTRY
+ARG DOCKER_UPSTREAM_REGISTRY
```

Comment:
should these have default values?

Reply:
If they're not given, the Docker default is to just use the latest from docker.io. We could provide defaults, but the implicit defaults are also okay.

Reply:
huh, i wouldn't have thought including the `:` but not a tag following it would just use latest.

Reply:
You're right about tag, added a default.